lib/katello/tasks/delete_orphaned_content.rake

```diff
@@ -16,6 +16,11 @@ namespace :katello do
 
   def remove_orphan(proxy)
     ForemanTasks.async_task(Actions::Katello::CapsuleContent::RemoveOrphans, proxy)
+    if proxy.pulp3_enabled? && proxy == SmartProxy.pulp_master
+      ForemanTasks.async_task(
+        Actions::Pulp3::Orchestration::Repository::RemoveOrphans, proxy)
+    end
```

Comment:
would it make sense to put this logic inside of Actions::Katello::CapsuleContent::RemoveOrphans ?

Reply:
I wasn't sure: I could see that it could, but it's not just for capsule operations. I am totally OK with moving it.

Reply:
i think we should move it mainly to reduce the # of tasks that are kicked off